client/styles/abstracts/_placeholders.scss

```diff
@@ -174,13 +174,15 @@
 		color: getThemifyVariable('inactive-text-color');
 		cursor: pointer;
 		& g, & path {
-			fill: getThemifyVariable('inactive-text-color');
+			// fill: getThemifyVariable('inactive-text-color');
+			fill: #fff;
```

Comment:
We use these themify variables because we have three different themes (light, dark, and contrast).  In the contrast theme the logo is black instead of white.

![image](https://github.com/processing/p5.js-web-editor/assets/28965286/a6efbd80-931a-4b7c-b504-cb7ee78b9b18)